docs/docs/index.mdx

```diff
@@ -1,25 +1,68 @@
 ---
 id: index
-title: Welcome to Ory!
+title: The Open Source Identity Platform
 slug: /
 sidebar_label: Welcome
 ---
 
-Ory is an open source ecosystem and a cloud offering authentication,
-authorization, access control, and delegation (OAuth2 & OpenID Connect) services
-and APIs:
-
-- [Ory Cloud](https://console.ory.sh) is a cloud service built on top of Ory
-  Open Source.
-- Ory Open Source offers scalable and hardened security software:
-- [Ory Kratos](https://github.com/ory/kratos)
-- [Ory Hydra](https://github.com/ory/hydra)
-- [Ory Oathkeeper](https://github.com/ory/oathkeeper)
-- [Ory Keto](https://github.com/ory/keto)
-
-Some parts of Ory are actively being developed. To learn more about roadmap and
-planned features head over to [early access](./early-access.mdx) and
-[project maturity](./ecosystem/versioning.md).
+Ory is the most complete and advanced open source platform offering a wide range
+of modern tools solving authentication, authorization, access control,
+application network security, and delegation. Ory implements a variety of
+best-practice standards including OAuth 2.0 / OAuth 2.1, OpenID Connect, Zero
+Trust Networking, Zanzibar Policy Framework, FIDO2 U2F, WebAuthn, TOTP, and many
+more.
+
+- [Ory Kratos](https://github.com/ory/kratos) is a fully customizable, API-only
+  platform for login, two-factor authentication, social sign in, passwordless
+  flows, registration, account recovery, email / phone verification, identity
+  and user management.
+- [Ory Hydra](https://github.com/ory/hydra) is an API-only OAuth 2.0 and OpenID
+  Connect provider that can interface with any identity and user management
+  system (e.g. Ory Kratos, Firebase, your PHP app, LDAP, SAML ...).
+- [Ory Oathkeeper](https://github.com/ory/oathkeeper) is a zero trust networking
+  proxy and sidecar for popular ingress services and API gateways. It checks if
+  incoming network request are authenticated and allowed to perform the
+  requested action.
+- [Ory Keto](https://github.com/ory/keto) is the world's first implementation of
+  [Google's Zanzibar](https://research.google/pubs/pub48190/) research paper, an
+  infinitely scalable and blazing fast authorization and permission service.
+  Think RBAC on globally distributed steroids.
+
+The best place to run Ory software is in [Ory Cloud](https://www.ory.sh/). Ory
+Cloud takes care of scalability, reliability, latency, availability, security,
+and risk management for your authentication and authorization needs. Running Ory
+software on Ory Cloud is as easy as:
+
+```
```

Comment:
Just some thoughts that I think might help with improving this section.

Could we maybe have a simple diagram here as well showing how each product fits in to the whole AuthN and AuthZ stack + the potential users' stack? This might also clarify immediately what people need.

We can also answer questions immediately through diagrams and simple language, such as:
- Is the Cloud offering just Kratos? 
- Is the cloud all of these listed? 
- Or is the cloud a separate thing?
- Can I implement my own branding?
- Is it "compatible" with JavaScript? (dumb question, but people might not fully get "api only" or "api first". We might need to make that more clear.
- Do I need to pay anything to start using these?